tools/README.md

```diff
@@ -98,13 +98,27 @@ bin/prepare_prebuilt_workers \
      -r containers/pre_built_workers
 ```
 
-This builds `cxx` and `go` images contain workers built from the commit/branch
-we wish to test.
+This builds `cxx` and `go` images contain workers built from the specific
+commit/branch we wish to test.
+
+You may also specify the repo where the source code is pulled from by specifying
+the repo, the following command would pull the source code from a forked repo
+to build `cxx` and `go` images.
+
+```shell
+bin/prepare_prebuilt_workers \
+     -l cxx:<user-name>/grpc:master \
```

Comment:
Here I would use "cxx:${USER}/grpc:master" and "go:${USER}/grpc-go:master" to make the examples more concrete.